modular_citadel/code/modules/vore/eating/vorepanel_vr.dm

```diff
@@ -651,7 +657,7 @@
 
 		if(failure_msg)
 			alert(user,failure_msg,"Error!")
-			return FALSE
+			return 0
```

Comment:
Why replace all instances of TRUE/FALSE with 1/0?

Reply:
Because having them as TRUE/FALSE played merry hell at times. With just 1/0 they work as intended with no glitches.